lib/src/components/form/form.dart

```diff
@@ -1374,12 +1384,28 @@ class FormField<T> extends StatelessWidget {
             child: Column(
               crossAxisAlignment: CrossAxisAlignment.stretch,
               children: [
-                mergeAnimatedTextStyle(
-                  style: error != null
-                      ? TextStyle(color: theme.colorScheme.destructive)
-                      : null,
-                  child: label.textSmall(),
-                  duration: kDefaultDuration,
+                Padding(
+                  padding: padding!,
+                  child: Row(
+                    mainAxisAlignment: labelAxisAlignment!,
+                    children: [
+                      Row(
+                        children: [
+                          if (leadingLabel != null) leadingLabel!,
+                          leadingGap!,
+                          mergeAnimatedTextStyle(
+                            style: error != null
+                                ? TextStyle(
+                                    color: theme.colorScheme.destructive)
+                                : null,
+                            child: label.textSmall(),
+                            duration: kDefaultDuration,
+                          ),
+                        ],
+                      ),
+                      if (trailingLabel != null) trailingLabel!,
+                    ],
+                  ),
```

Comment:
Can we simplify this in 1 row instead of 2 nested rows? or is there something I missed?